src/applications/hca-v2-beta/config/form.js

```diff
@@ -582,6 +582,7 @@ const formConfig = {
                 confirmationCode: response.data.attributes.guid,
                 size: file.size,
               }),
+              gaEventName: 'document-uploaded',
```

Comment:
Do we need this customization option? Could we use `file-uploaded` or `document-uploaded` everywhere?

Reply:
Chatted w/ @nedierecel -- I'll make it app-unique but non-customizable:
`{ event: '<app->file-uploaded' }`

Reply:
Thanks @tlei123 - I will also make sure Google Tag Manager settings look for events that end in `file-uploaded`